client/components/upgrades/credit-card-form/style.scss

```diff
@@ -61,28 +54,7 @@
 		font-size: 11px;
 		font-weight: bold;
 		position: absolute;
-			left: 13px;
-			top: 12px;
-
-		.error {
-			color: $alert-red;
-			margin-left: 1em;
-		}
-	}
-
-	&.focus {
```

Comment:
No idea what these styles were meant to do. I mean, I _do_, but it doesn't seem like they've actually been used.